woocommerce/woocommerce
# CYS: Improve patterns order

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:






This PR improves the pattern order for the `Intro`, `About`, and `Services` categories.

### Intro

| Woo Assembler | DotCom Assembler |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/ef8e1c6d-2a60-482f-a6c7-c12d1c58fa0e) | ![image](https://github.com/user-attachments/assets/b0912f33-ed22-45e7-babb-44582af6fcc4)| 

### About

| Woo Assembler | DotCom Assembler |
|--------|--------|
|![image](https://github.com/user-attachments/assets/6bb0d252-01d8-4523-87bb-fcb39cc14cbb) | ![image](https://github.com/user-attachments/assets/8e916a97-32b7-42ec-a557-286de6bff1af)

### Services

| Woo Assembler | DotCom Assembler |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/d5a3934f-9b16-4e00-9b04-765ada807f58) |  ![image](https://github.com/user-attachments/assets/f89cdb8b-4e6f-4af4-9dc5-4e33f3c0db41)

 
 




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed.
3. Head over to Tools > WCA Test Helper > Features.
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
11. Click Desing your Homepage.
12. For the `Intro,` `About,` and `Services` categories, ensure that the patterns are listed like the image in left columns.




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Improve patterns order

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
